src/main/webapp/i18n/de/global.json

```diff
@@ -94,7 +94,9 @@
                 "jhipster-needle-menu-add-admin-element": "JHipster will add additional menu entries here (do not translate!)"
             },
             "language": "Sprache",
-            "noNotifications": "Keine neuen Mitteilungen verfügbar",
+            "guidedTour": "Anleitungstour",
```

Comment:
Anleitungstour sounds very generic. I think using "Guided Tour" as German Description is just fine. There is also a german wikipedia article about guided tours: https://de.wikipedia.org/wiki/Guided_Tour